doc/licenses/spring-web-5.3.33/README

```diff
@@ -1,7 +1,7 @@
 Spring Framework (https://spring.io/projects/spring-framework)
 --------------------------------------------------------------
 
-    Version: 5.3.25
+    Version: 5.3.33
```

Comment:
[Dependabot ](https://github.com/apache/guacamole-client/security/dependabot/64) recommends 6.0.18. Why not use that version?

Reply:
I ran across this when working #972 - I believe it bumps minimum Java version to 17, and I'm reasonably certain it also requires the move to jakarta.* namespaces.

Reply:
Ah ok. Makes sense. It won't make dependabot fully happy, but I guess it's the best we can do.

Reply:
I think we can avoid Spring entirely here - I see it used in guacamole-auth-duo only for `UriComponentsBuilder`:

```java
String builtUrl = UriComponentsBuilder
        .fromUriString(redirectUrl)
        .queryParam(Credentials.RESUME_QUERY, DuoAuthenticationProvider.PROVIDER_IDENTIFER)
        .build()
        .toUriString();
```

I _think_ this can be rewritten to just use `UriBuilder`: https://javaee.github.io/javaee-spec/javadocs/javax/ws/rs/core/UriBuilder.html